.eslintrc.js

```diff
@@ -0,0 +1,321 @@
+module.exports = {
+    'env': {
+        'browser': true,
+        'es6': true,
+        'node': true,
+        'jest': true
+    },
+    'extends': [
+        'eslint:recommended',
+        'plugin:react/recommended',
+        'airbnb'
+    ],
+    'globals': {
+        'Atomics': 'readonly',
+        'SharedArrayBuffer': 'readonly',
+        "shallow": true,
+        "render": true,
+        "mount": true
+    },
+    'parserOptions': {
+        'ecmaFeatures': {
+            'jsx': true
+        },
+        'ecmaVersion': 2018,
+        'sourceType': 'module'
+    },
+    'plugins': ['react'],
+    settings: {
+        react: {
+            version: 'detect'
+        },
+        createClass: 'createReactClass',
+        pragma: 'React',
+        flowVersion: '0.53',
+        'import/resolver': {
+            node: {
+                moduleDirectory: ['node_modules', 'src'],
+            },
+        },
+    },
+    'rules': {
+        'accessor-pairs': 'error',
```

Comment:
All of these rules shouldn't need to be enabled. We can turn off specific rules. It should extend airbnb's guide